build.fsx

```diff
@@ -348,17 +348,31 @@ Target "Prepare" DoNothing
 // BUILD
 // --------------------------------------------------------------------------------------
 
-Target "BuildMain" (fun _ -> build !! "MathNet.Numerics.sln")
+let dotnetBuild solution = DotNetCli.Build (fun p ->
+        { p with
+            Project = solution
+            Configuration = "Release" })
+
+Target "BuildMain" (fun _ -> dotnetBuild "MathNet.Numerics.sln")
 Target "BuildSigned" (fun _ -> buildSigned !! "MathNet.Numerics.sln")
+Target "RestoreMain" (fun _ -> DotNetCli.Restore (fun p ->
+        { p with
+            Project = "MathNet.Numerics.sln"
+            NoCache = true }))
```

Comment:
What's the rationale behind `NoCache=true`?

Reply:
This was just to be on the safe side for starters. IMHO, once the project structure is stabilised the cache should be enabled.